example/golack/eventsapi/main.go

```diff
@@ -52,9 +52,27 @@ func (r *Receiver) Receive(wrapper *eventsapi.EventWrapper) {
 			return
 		}
 
-		message := webapi.NewPostMessage(typed.ChannelID, echoMsg)
-		message.AsUser = true
-		log.Printf("Payload: %+v", message)
+		message := webapi.NewPostMessage(typed.ChannelID, echoMsg).
```

Comment:
The output is as below:

![image](https://user-images.githubusercontent.com/1629963/87844102-bc99e880-c8f4-11ea-983a-98b03587bb72.png)